tunnel/intercept/svcpoll.go

```diff
@@ -105,47 +109,65 @@ func diffServices(edgeServices []edge.Service, knownServices map[string]edge.Ser
 	return
 }
 
-func updateServices(context ziti.Context, interceptor Interceptor, resolver dns.Resolver, added, removed, all map[string]edge.Service) {
+func updateServices(context ziti.Context, interceptor Interceptor, resolver dns.Resolver, added, removed, all map[string]*entities.Service) {
 	log := pfxlog.Logger()
 	for _, svc := range added {
 		if stringz.Contains(svc.Permissions, "Dial") {
+			clientConfig := &entities.ServiceConfig{}
+			if found, err := svc.GetConfigOfType(entities.ClientConfigV1, clientConfig); found && err == nil {
+				svc.ClientConfig = clientConfig
+			} else if !found {
```

Comment:
I wonder if these messages will make folks think something is broken when it really isn't?

For the `!found` case, would it be reasonable to log at Debug here and let the interceptor log an error as appropriate?

For `found && err !=nil`, it seems like this should be logged as Error and the loop immediately continues?

Reply:
Makes sense. If it can't find the client config, should we let it try to host it?

Reply:
Oh, I had to think about why you asked this question... I was originally thinking that the configs were all grabbed at once, but the server config is grabbed separately below/. So you're saying that an error here reading the client config shouldn't continue the loop. Agreed.